package-lock.json

```diff
@@ -35,7 +35,7 @@
         "eslint-config-prettier": "^9.1.0",
         "eslint-plugin-next-on-pages": "^1.11.2",
         "eslint-plugin-prettier": "^5.1.3",
-        "framer-motion": "^11.2.10",
```

Comment:
i dont think the minor version would of made a big diff. But thnx for being cautious